### PR TITLE
Add ndmflowAggregator to fakeintake and add new-e2e ndm/netflow test.

### PR DESCRIPTION
**Pull request by jmw51798**
### What does this PR do?
Add ndmflowAggregator to fakeintake and add new-e2e ndm/netflow test.

### CODE REVIEW

**Review by FlorianVeaux — APPROVED** [review]

File: test/fakeintake/aggregator/checkRunAggregator.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/DataDog/datadog-agent/test/fakeintake/api"
 )
 
-//nolint:revive // TODO(APL) Fix revive linter
+// CheckRun represents a check run payload
```
Comment (pducolin) [inline test/fakeintake/aggregator/checkRunAggregator.go:16]: **👏 praise**
Thank you!

File: test/fakeintake/client/client_test.go [hunk 2]
```diff
@@ -497,4 +500,16 @@ func TestClient(t *testing.T) {
 			assert.Equal(t, expectedHostname, p.Hostname)
 		}
 	})
+
+	t.Run("getNDMFlows", func(t *testing.T) {
```
Comment (pducolin) [inline test/fakeintake/client/client_test.go:504]: **💬 suggestion**
You can test the exported `GetNDMFlows` to ensure the external API is correct

**Review by pducolin — APPROVED** [review]
Well done!

File: test/fakeintake/aggregator/ndmflowAggregator.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```
Comment (pducolin) [inline test/fakeintake/aggregator/ndmflowAggregator.go:1]: **❓ question**
Can you add your team to the owners of this file?

**Review by pducolin — APPROVED** [review]

File: test/fakeintake/client/client_test.go (hunk 2 above)
Comment (jmw51798) [reply test/fakeintake/client/client_test.go:504]: Good suggestion - done.

File: test/fakeintake/aggregator/ndmflowAggregator.go (hunk 3 above)
Comment (jmw51798) [reply test/fakeintake/aggregator/ndmflowAggregator.go:1]: Yes - done.

**Pull request merged**